plugins/available/jump.plugin.bash

```diff
@@ -1,9 +1,9 @@
 cite about-plugin
-about-plugin 'initialize jump (see https://github.com/gsamokovarov/jump)'
+about-plugin 'initialize jump (see https://github.com/gsamokovarov/jump). Add `export JUMP_OPTS="--bind=z"` to change keybinding'
```

Comment:
Since `JUMP_OPTS` is actually as an array, wondering if we want `JUMP_OPTS=("--bind=z")` in the example?

They both work since there's only one argument being passed, but might be good to set the `OPTS variable as array` pattern in case we start using OPTS variables in more plugins.

Reply:
I had thought of that and yes, makes sense. Since we are doing array expansion in the plugin script, makes sense to be explicit with the example declaration.